sdks/python/apache_beam/runners/direct/evaluation_context.py

```diff
@@ -341,5 +335,6 @@ def get_keyed_state(self, key):
     if not self.existing_keyed_state.get(key):
       self.existing_keyed_state[key] = DirectUnmergedState()
     if not self.partial_keyed_state.get(key):
-      self.partial_keyed_state[key] = self.existing_keyed_state[key].clone()
+      self.partial_keyed_state[key] = (
+          self.existing_keyed_state[key].custom_copy())
```

Comment:
<!--thread_id:cc_137811815_t; commit:8650ea65963f8e3efc5555d8b3aa5474292efd20; resolved:1-->
<!--section:context-quote-->
> **charlesccychen** wrote:
> Let's just rename this to `clone()`.

<!--section:body-->
I preferred to avoid that name, as it was not really a deep copy. I changed it to copy().